packages/utils/src/internal/schemas/address.ts

```diff
@@ -0,0 +1,12 @@
+import { z } from "zod";
+import { usStateSchema } from "@metriport/api-sdk/medical/models/common/us-data";
+import { defaultOptionalString, defaultZipString } from "./shared";
+
+export const addressStrictSchema = z.object({
+  addressLine1: z.string().min(1),
+  addressLine2: defaultOptionalString,
+  city: z.string().min(1),
+  state: usStateSchema,
+  zip: defaultZipString,
+  country: z.literal("USA").default("USA"),
+});
```

Comment:
now that you're adding this here, could you possibly go back to #3131 and use this to simplify more complicated omit type i suggested?

Reply:
This is not part of the final commit of this PR, maybe you saw this when reviewing individual ones?